new-log-viewer/src/utils/actions.ts

```diff
@@ -70,39 +68,43 @@ const EDITOR_ACTIONS : ActionType[] = [
 /* eslint-enable sort-keys */
 
 /**
- * Handles an action based on the given action name, log event number, and total number of events.
+ * Handles an action based on the given action name.
  *
  * @param actionName
- * @param logEventNum
- * @param numEvents
+ * @param pageNum
+ * @param numPages
+ * @param loadPage
  */
-const handleAction = (actionName: ACTION_NAME, logEventNum: number, numEvents: number) => {
-    const pageSize = getConfig(CONFIG_KEY.PAGE_SIZE);
+const handleAction = (
```

Comment:
To avoid creating this extra handler -

Can we change loadPage to have it take in either a `newPage` number or ACTION_NAME? if `typeof newPage` is ACTION_NAME, we calculate the new page number there and proceed to the other logic.

Reply:
If `loadPage` can handle ACTION_NAME.XXXX_PAGE directly, we can avoid passing the pageNum, the numPages and the loadPage callback all the way here, which might be a bit cleaner. What do you think?

> Is it being used anywhere else?

It's also used by the editor shortcuts if i'm not wrong.